src/SourceTerm/PointSource.cpp

```diff
@@ -176,15 +185,24 @@ void seissol::sourceterm::addTimeIntegratedPointSourceNRF( real const i_mInvJInv
 
 void seissol::sourceterm::addTimeIntegratedPointSourceFSRM( real const i_mInvJInvPhisAtSources[tensor::mInvJInvPhisAtSources::size()],
                                                             real const i_momentTensor[tensor::momentFSRM::size()],
+                                                            real const i_velocityComponent[3],
                                                             PiecewiseLinearFunction1D const* i_pwLF,
                                                             double i_fromTime,
                                                             double i_toTime,
                                                             real o_dofUpdate[tensor::Q::size()] )
 {
+  real moment[tensor::momentFSRM::size()];
+  for(int i = 0; i < 6; i++)
+    moment[i] = i_momentTensor[i];
+  for(int i = 0; i < 3; i++)
+    moment[6+i] = i_velocityComponent[i];
+  for(unsigned i = 9; i < tensor::momentFSRM::size(); i++)
+    moment[i] = 0;
+
```

Comment:
Again, if you simply copy velocityComponent in i_momentTensor in transformMomentTensor, then you don't need these lines of code.